oidc-ui/src/services/walletService.js

```diff
@@ -9,6 +9,7 @@ const modalityIconPath = {
   WALLET: "images/wallet_icon.svg",
   BIO: "images/bio_icon.svg",
   PWD: "images/sign_in_with_otp.png",
+  KBA: "images/sign_in_with_otp.png",
```

Comment:
why same image?

Reply:
Created task to create new logo https://mosip.atlassian.net/browse/ES-626